docs/compilers/CSharp/CommandLine.md

```diff
@@ -50,7 +50,8 @@
 | `/checked`{`+`&#124;`-`} | Generate overflow checks
 | `/unsafe`{`+`&#124;`-`} | Allow 'unsafe' code
 | `/define:`*symbol list* | Define conditional compilation symbol(s) (Short form: `/d`)
-| `/langversion`:*string* | Specify language version mode: `ISO-1`, `ISO-2`, `3`, `4`, `5`, `6`, `7`, `7.1`, `7.2`, `Default` (latest major version), or `Latest` (latest version, including minor versions)
+| `/langversion:?` | Display the allowed values for language version
+| `/langversion`:*string* | Specify language version such as `default` (latest major version), or `latest` (latest version, including minor versions)
```

Comment:
Never mind, found it.

---
In reply to: [121556359](https://github.com/dotnet/roslyn/pull/20153#discussion_r121556359) [](ancestors = 121556359)